dist/images/vpcnatgateway/nat-gateway.sh

```diff
@@ -13,8 +13,14 @@ function exec_cmd() {
 }
 
 function init() {
-    lanCIDR=$1
+    # run once is enough
+    iptables-save | grep DNAT_FILTER && exit 0
+    # add static chain
+    # this also a flag to make sure init once
+    iptables -t nat -N DNAT_FILTER
     ip link set net1 up
+    ip link set dev net1 arp off
```

Comment:
if nic net1 arp already on，but nic net1 has no eip and no route，net1 will response all arp request， this will cause problems。

我们测试发现，net 1 网卡未配置ip且未配置到公网路由的时候，net1会影响所有arp广播，导致所有eip都无法使用

Reply:
有极小概率，nat pod会停在init之后，比如恰好init之后，kube-ovn-controller崩溃，切换，此时如果arp on，就会响应arp，此时所有eip都会受影响。